test/color/test_rgb.py

```diff
@@ -2,306 +2,237 @@
 
 import kornia
 import kornia.testing as utils  # test utils
+from kornia.testing import BaseTester
 
 import torch
 from torch.autograd import gradcheck
 from torch.testing import assert_allclose
 
 
-class TestRgbToRgba:
-    def test_smoke(self, device):
-        data = torch.rand(3, 4, 4).to(device)
-        assert kornia.rgb_to_rgba(data, 0.).shape == (4, 4, 4)
+class TestRgbToBgr(BaseTester):
+    def test_smoke(self, device, dtype):
+        C, H, W = 3, 4, 5
+        img = torch.empty(C, H, W, device=device, dtype=dtype)
+        assert kornia.rgb_to_bgr(img) is not None
+        assert kornia.color.rgb_to_bgr(img) is not None
```

Comment:
Just because the output `is not None`, does not mean it is a `torch.Tensor`.

```suggestion
        assert isinstance(kornia.rgb_to_bgr(img), torch.Tensor)
        assert isinstance(kornia.color.rgb_to_bgr(img), torch.Tensor)
```

Reply:
gotcha. Will fix